src/coral_models/prepare_raw_data.py

```diff
@@ -0,0 +1,475 @@
+                read_aloud_data.loc[row_i, "sentence_id"] = sentence_content_to_id[
+                    row["transcription"]
+                ]
+            else:
+                sentence_content_to_id[row["transcription"]] = len(
+                    sentence_content_to_id
+                )
+                read_aloud_data.loc[row_i, "sentence_id"] = sentence_content_to_id[
+                    row["transcription"]
+                ]
+
+        # Make a recorder_id. This is not in the read aloud data, as no we have
+        # no information about the recorders for the read aloud data.
+        read_aloud_data["recorder_id"] = -1
+
+        # Drop columns that are not in the recording metadata
+        read_aloud_data = read_aloud_data[recording_metadata.columns]
+
+        # Append to recording metadata list
+        recording_metadata_list.append(read_aloud_data)
+
+    # Concatenate all recording metadata
+    all_recording_metadata = (
+        pd.concat(recording_metadata_list, axis=0)
+        .drop_duplicates()
+        .reset_index(drop=True)
+    )
+
+    # Make a recording id column
+    all_recording_metadata["recording_id"] = "r" + all_recording_metadata.index.astype(
+        str
+    )
+
+    # We have updated the sentence_content_to_id dict, so we also need to update the
+    # sentences dataframe with the new sentence ids
+    sentences = (
+        pd.DataFrame.from_dict(sentence_content_to_id, orient="index")
+        .reset_index()
+        .rename(columns={"index": "text", 0: "sentence_id"})
+    )
+
+    # Prepend the sentence id column with "s"
+    sentences["sentence_id"] = "s" + sentences.index.astype(str)
+    all_recording_metadata["sentence_id"] = "s" + all_recording_metadata[
+        "sentence_id"
+    ].astype(str)
+
+    # Start and Stop columns are in the format "HHMM-DD-MM-YY" and
+    # "DD/MM/YYYY HH:MM:SS" and need to be converted to
+    # "DD/MM/YYYY HH:MM:SS+02:00"
+    all_recording_metadata["start"] = all_recording_metadata["start"].apply(
+        correct_timestamp
+    )
+    all_recording_metadata["stop"] = all_recording_metadata["stop"].apply(
+        correct_timestamp
+    )
+
+    # The dimensions column has values in the format "X,Y,Z" and "XxYxZ". These need to
+    #  be converted to "X,Y,Z"
+    all_recording_metadata["dimensions"] = all_recording_metadata["dimensions"].apply(
+        lambda x: ",".join(x.split("x"))
+    )
+
+    # The noise column has both "none", "ingen", "trafik", "traffic", which needs to be
+    # only "none" or "traffic".
+    all_recording_metadata["background_noise_level"] = all_recording_metadata[
+        "background_noise_level"
+    ].apply(lambda x: dict(ingen="none", trafik="traffic").get(x, x))
+
+    return all_recording_metadata, sentences
+
+
+def prepare_raw_data(
+    input_path_str: str = "data/raw",
+    output_path_str: str = "data/processed",
+    metadata_path_str: str = "data/raw/metadata.csv",
```

Comment:
Using strings like this makes it raise an error on Windows machines (they don't use forward slashes). Maybe change them to `Path` objects instead, which respect the OS?